cmd/traffic_layout/engine.cc

```diff
@@ -0,0 +1,290 @@
+/** @file
+
+  A brief file description
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// for engine of traffic runroot
+// function introduction in engine.h
+
+#include "ts/runroot.h"
+#include "ts/I_Layout.h"
+#include "ts/ink_error.h"
+#include "ts/ink_args.h"
+#include "ts/I_Version.h"
+#include "ts/ink_file.h"
+
+#include "engine.h"
+#include "file_system.h"
+
+#include <fstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
I know we don't have a good standard here, but I personally much prefer to *not* do this. I.e. I find this a lot more readable, and safer, than what we have in here:

```
std::string input

std::cin >> input;
```

etc. yes, I know it's a little more verbose, but it is now crystal clear what exactly you mean with "string" and "cin" here, both of which are very generic names.

Reply:
Yes, I agree that `using namespace std;` is not a good idea in our context.

Reply:
I have no issue with this